internal/services/sentinel/sentinel_watchlist_item_resource.go

```diff
@@ -184,29 +165,30 @@ func (r WatchlistItemResource) Update() sdk.ResourceFunc {
 				return fmt.Errorf("decoding %+v", err)
 			}
 
-			watchlistId, err := parse.WatchlistID(model.WatchlistID)
+			watchlistId, err := watchlistitems.ParseWatchlistItemID(model.WatchlistID)
```

Comment:
This is causing a test failure. I think it wants to be:

```suggestion
			watchlistId, err := watchlistitems.ParseWatchlist(model.WatchlistID)
```